CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.2.z]
 
+## [1.2.1] - 2024-11-05
```

Comment:
Would it be possible to make this release today?

Reply:
Sure, this was moving faster than I thought. Updated the date to today too.